### DISCUSSION TITLE
Custom Log Intrgration

### DISCUSSION THREAD

**Discussion by larry1788** [Security-Onion-Solutions/securityonion#12244] [Unsupported Versions]
Hello Security Onion expert
     How to use Security Onion System's Custom Log Intrgration to create an exclusive LOG in the management system. I have referred to the tutorials on the Elastic website, but without success. Where can I get step-by-step tutorials?

2024-01-23 08:37:58,163 fail2ban.filter         [676]: INFO    [nginx-modsec] Found 10X.10X.14X.1 - 2024-01-23 08:37:58

**Comment by cm-ops** [comment]
You would use the `custom logs` integration in Fleet. Also, https://docs.elastic.co/integrations/log
